collect_app/src/main/java/org/odk/collect/android/widgets/OSMWidget.java

```diff
@@ -169,11 +168,7 @@ private void launchOpenMapKit() {
 
             //launch activity if it is safe
             if (isIntentSafe) {
-                // notify that the form is waiting for data
-                waitForData();
-
-                // launch
-                ((Activity) ctx).startActivityForResult(launchIntent, RequestCodes.OSM_CAPTURE);
+                startActivityForResult(launchIntent, RequestCodes.OSM_CAPTURE, -1);
```

Comment:
These -1 magic numbers are confusing. Is there a constant you could use? Alternately, perhaps it's worth introducing a signature without that third parameter.

Reply:
:+1: